tests/system_tests_distribution.py

```diff
@@ -3826,6 +3823,7 @@ def __init__(self,
         self.error                = None
         self.messages_per_sender  = 100
```

Comment:
Non-patch observation: is 100 messages *really* necessary?  CI systems tend to be painfully slow.  Can the same degree of confidence that this works be achieved by say 20 messages?

Reply:
I am curious to find out why this test is failing. I am sure the failure has nothing to do with the number of messages sent. I want to push this logging change to main and analyze the debug output when the test fails.